### ISSUE TITLE
Collapse into node10-express repo

### ISSUE THREAD

**Issue by alexellis** [openfaas-incubator/node10-express-service#2]
## Task

Move all code into the [node10-express-template](https://github.com/openfaas-incubator/node10-express-template) repo and the examples into the README.

Archive or delete repo after this has been completed.

**Comment by alexellis** [comment]
@burtonr is this something you could take on?

**Comment by burtonr** [comment]
Absolutely! Just add it as another template in that same repo correct?

I can get that done this evening

**Comment by alexellis** [comment]
Yes, include all the examples in the README please and update the docs repo GitHub.com/openfaas/docs

**Comment by burtonr** [comment]
The migration is complete (see the linked PR above).

I'll wait for the other node docs update PR to get merged (it should be ready to merge) to avoid having to deal with the conflict
https://github.com/openfaas/docs/pull/212

**Comment by alexellis** [comment]
Sure, that's been done now @burtonr
